pkg/network/node/ovscontroller.go

```diff
@@ -76,7 +76,8 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 		return err
 	}
 	_ = oc.ovs.DeletePort(Vxlan0)
-	_, err = oc.ovs.AddPort(Vxlan0, 1, "type=vxlan", `options:remote_ip="flow"`, `options:key="flow"`)
+	_, err = oc.ovs.AddPort(Vxlan0, 1, "type=vxlan", `options:remote_ip="flow"`, `options:key="flow"`,
```

Comment:
This is not the tun0 interface. Wrong line?

Reply:
Yes, seems like wrong line. 
tun0 port is on line 85.

Reply:
Bah, thanks.  Doing too many things at once :-(